src/contentmanager.cpp

```diff
@@ -272,8 +275,8 @@ void ContentManager::eraseBook(const QString& id)
     tabBar->closeTabsByZimId(id);
     kiwix::Book book = mp_library->getBookById(id);
     QString dirPath = QString::fromStdString(kiwix::removeLastPathElement(book.getPath()));
-    QString filename = QString::fromStdString(kiwix::getLastPathElement(book.getPath())) + "*";
-    eraseBookFilesFromComputer(dirPath, filename);
```

Comment:
It is probably to `eraseBookFilesFromComputer` to split the bookPath and erase book properly.

The signature of the method should probably be:
`bool eraseBookFilesFromComputer(const Book& book)`